subprojects/parseq-guava-interop/src/test/java/com/linkedin/parseq/guava/ListenableFutureUtilTest.java

```diff
@@ -62,6 +62,13 @@ public void testFromListenableFuture() throws Exception {
     Assert.assertTrue(task.isDone());
     Assert.assertTrue(task.isFailed());
     Assert.assertEquals(task.getError().getCause().getClass(), CancellationException.class);
+
+    listenableFuture = new ListenableFutureUtil.SettableFuture<>();
```

Comment:
I don't understand this change, did line 39-43 already test this path? Only difference is that there is `runUntilComplete(task)`